miqcli/collections/instances.py

```diff
@@ -44,10 +44,13 @@ class Collections(CollectionsMixin):
                   help='vendor of an instance(s)')
     @click.option('--itype', type=str, default='',
                   help='type of an instance(s) - ex. "Openstack", "Amazon"...')
+    @click.option('-v', '--verbose', count=True,
```

Comment:
Why are we introducing a new option for `verbose`? Can we use the verbose value that the user can set at the top level of the CLI? i.e. `$ miqcli --verbose <command> <sub-command> <options>`